### PR TITLE
Add apply button to hero section

### LINKED ISSUE #555: ScholarX 2020: Add apply button to the hero section
**Is your feature request related to a problem? Please describe.**
There should be a button at the hero section with the text "APPLY" It should be linked to the apply section of the same page. 

## use `scholarx-2020` branch

### PR DESCRIPTION
**Pull request by 99sumedha**
## Purpose
The purpose of this PR is to fix #555 
Move the page to the apply section when clicked
## Goals

Now a user can easily move to the apply section straight forward

## Approach

Add an id to the apply section and link it to the button in the hero section

### Preview Link

https://pr-560-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/scholarx/archive/2019/index.html [hunk 1]
```diff
@@ -173,9 +173,10 @@ <h5 class="mb-0">
         <div class="row justify-content-center">
             <!--Profile cards goes here-->
             <div id="teamContent"></div>
-
+            <div class="row justify-content-center">
             <button id="btnShowMore" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
         </div>
+        </div>
     </div>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/archive/2019/index.html]: Do not change other files

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Please remove `class="pb-5"`  on line number 45.
```
<p class="pb-5">ScholarX is an exclusive 6-month program aimed at providing
                            mentoring support to a selected pool of ...
```

**Comment by 99sumedha** [comment]
class="pb-5" class removed

File: src/main/webapp/scholarx/archive/2019/index.html (hunk 1 above)
Comment (jayasanka-sack) [reply src/main/webapp/scholarx/archive/2019/index.html]: @99sumedha Do not change this file. We need the apply button only for 2020 page

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Comment by 99sumedha** [comment]
New commit added

File: src/main/webapp/scholarx/2020/index.html [hunk 2]
```diff
@@ -303,7 +304,6 @@ <h5 class="mb-0">
         <p class="display-3 text-center">Mentors</p>
         <!--Profile cards goes here-->
         <div id="teamContent"></div>
-        <div class="row justify-content-center">
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html:111]: Do not remove this line

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/scholarx/2020/index.html [hunk 3]
```diff
@@ -302,8 +303,8 @@ <h5 class="mb-0">
     <div class="container">
         <p class="display-3 text-center">Mentors</p>
         <!--Profile cards goes here-->
-        <div id="teamContent"></div>
         <div class="row justify-content-center">
+        <div id="teamContent"></div>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html:112]: What is this?

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/scholarx/2020/index.html [hunk 4]
```diff
@@ -107,8 +108,8 @@ <h1 class="mb-0 text-onelive font-weight-bold text-center">Program Timeline</h1>
     <div class="container">
         <p class="display-3 text-center">Mentors</p>
         <!--Profile cards goes here-->
-        <div id="teamContent"></div>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html:110]: do not remove this line

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Remove <<<<<<< HEAD
======= and >>>>>>> 19d1d643772e7487696ef0592233b54f0be664c6

File: src/main/webapp/scholarx/2020/index.html [hunk 5]
```diff
@@ -539,7 +540,8 @@ <h1 class="mb-0 text-onelive">Take a look at our past</h1>
     </div>
 </section>
 
-<section class="section pb-5" style="background : #e3f6fa">
+
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html]: Remove this whitespace

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/scholarx/2020/index.html (hunk 5 above)
Comment (99sumedha) [reply src/main/webapp/scholarx/2020/index.html]: Done

**Comment by jayasanka-sack** [comment]
Closing since we have to merge scholarx-2020 branch

**Pull request closed without merging**